tavern/internal/www/src/utils/enums.ts

```diff
@@ -16,6 +16,7 @@ export enum PageNavItem {
     tasks='Tasks',
     createQuest= 'Create new quest',
     hosts="Hosts",
+    users="User Management",
```

Comment:
We should change these to be related to admin settings since we expect more kinds of admin controls on this page in the future.